common/src/model_registration/register.py

```diff
@@ -145,8 +184,29 @@ def main(args):
     registered_model = ml_client.models.create_or_update(model)
     print(f"Model registered. AssetID : {registered_model.id}")
 
-    (Path(registration_details)).write_text(registered_model.id)
-    print("Saved model registration details in output text file.")
+    # Updating the description after registring (*Bugs need to be fixed)
```

Comment:
"Bugs need to be fixed" sounds scary. Can you elaborate a bit?

Reply:
@elesel issue with long model description it does not go in a single flow while adding a new model version. But update works in second trial